ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/bfd_multihop.schema.yml

```diff
@@ -0,0 +1,27 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  bfd_multihop:
+    documentation_options:
+      filename: Fabric Variables
+      table: BFD Settings
+    type: dict
+    description: BFD Multihop tuning
+    keys:
+      interval:
+        type: int
+        convert_types:
+          - str
+        default: 300
```

Comment:
we could add intervals

```
Leaf1(config-router-bfd)#multihop interval ?
  <50-60000>  Rate in milliseconds between 50-60000
Leaf1(config-router-bfd)#multihop interval 666 min-rx ?
  <50-60000>  Rate in milliseconds between 50-60000
Leaf1(config-router-bfd)#multihop interval 666 min-rx 666 multiplier ?
  <3-50>  Range is 3-50

Leaf1(config-router-bfd)#multihop interval 666 min-rx 666 multiplier 42
```

Reply:
addressed in [f94bf27](https://github.com/aristanetworks/ansible-avd/pull/2507/commits/f94bf272c4edf2aca1a925f2d2bf2469136eb45b)